.projenrc.ts

```diff
@@ -27,6 +27,32 @@ const project = new awscdk.AwsCdkConstructLibrary({
       },
     },
   },
+  pullRequestTemplateContents: [
+    `### Issue # (if applicable)
+
+Closes #<issue number here>.
+
+### Reason for this change
+
+<!--What is the bug or use case behind this change?-->
+
+### Description of changes
+
+<!--What code changes did you make? Have you made any important design decisions?-->
+
+### Description of how you validated changes
+
+<!--Have you added any unit tests and/or integration tests?-->
+
+### Checklist
+
+- [ ] My code adheres to the [CONTRIBUTING GUIDE](https://github.com/open-constructs/aws-cdk-library/blob/main/CONTRIBUTING.md)
```

Comment:
FWIW I don't think checklists provide much value. They become a box ticking exercise.

It's totally a good idea to call out the contribution guide though.